src/Raven.Client/Constants.cs

```diff
@@ -102,7 +102,12 @@ private Documents()
 
             public const int MaxDatabaseNameLength = 128;
 
-            public const string UnchangedSubscriptionsChangeVecotr = "DoNotChange";
+            public enum SubscriptionChagneVectorSpecialStates
+            {
```

Comment:
No, that's a way to feed studio's ui with values, a "SpecificChangeVector" value might confuse ClientAPI's users